renderlib/io/FileReaderZarr.cpp

```diff
@@ -330,7 +411,7 @@ FileReaderZarr::loadMultiscaleDims(const std::string& filepath, uint32_t scene)
         auto path = dataset["path"];
         if (path.is_string()) {
           std::string pathstr = path;
-          auto result = tensorstore::Open({ { "driver", "zarr" },
+          auto result = tensorstore::Open({ { "driver", tensorstoreZarrDriverName() },
```

Comment:
So tenserstore has its own implementation for the drivers for both Zarr v2 and v3?